local/linux/README.md

```diff
@@ -39,7 +39,7 @@ These playbooks will do the following in a local environment:
       ```
   * Install python dependencies
     ```bash
-    $ pip install six
+    $ pip install six==1.10.0 docker==2.4.2 ansible==2.3.0
```

Comment:
starting today, i was experiencing issues when issuing `pip install -U six docker ansible`.  Specifying these versions w/o a -U did the trick for me.  Let me know if you'd like to keep this commit or not.